client/src/components/basic/task-instructions/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Statistics from './components/Statistics';
+import Rank from './components/Rank';
+import styles from './task-instructions.module.scss';
+import './task-details.scss';
+
+interface ITaskInstructionProps {
+	title: string;
+	html: JSX.Element;
+	favorites: number;
+	rating: number;
+	createdBy: string;
+}
+
+const TaskInstructions: React.FC<ITaskInstructionProps> = (props: ITaskInstructionProps) => {
+	return (
+		<div className={styles.taskContainer}>
+			<div className={styles.header}>
+				<Rank difficulty={6} />
```

Comment:
You should not mock data like that. You need to mock all the data from container, component should just display the data